test/cucumber/step-definitions/margin.steps.ts

```diff
@@ -0,0 +1,65 @@
+import { Given, TableDefinition, Before } from 'cucumber';
+import ganache from 'ganache-core';
+import Web3 from 'web3';
+import { Account } from 'web3-core';
+import BN from 'bn.js';
+
+import abi from '../../../artifacts/development/abi/LiquidityPoolInterface.json';
+import deployment from '../../../artifacts/development/deployment.json';
+
+const web3 = new Web3(ganache.provider() as any);
+
+const poolAddress = deployment['pool'];
+const poolContract = new web3.eth.Contract(abi as any, poolAddress);
+
+let alice: Account;
+let bob: Account;
+
+Before(async () => {
+  alice = web3.eth.accounts.create();
+  bob = web3.eth.accounts.create();
+});
+
+const accountOf = (name: string): Account => {
+  let accounts: any = {
+    Alice: alice,
+    Bob: bob,
+  };
+  return accounts[name];
+}
+
+const transfer = async (name: string, amount: BN) => {
+  const accounts = await web3.eth.getAccounts();
+  await web3.eth.sendTransaction({ to: accountOf(name).address, from: accounts[0], value: amount});
+};
+
+const parseAmount = (amount: string): BN => {
+  const parsed = amount.replace(' ', '').replace('$', '').replace('_', '');
```

Comment:
For Laminar Chain implementation, `$` means the unit is dollar, i.e. 10 ** 18. so `$10` is 10**19.

Reply:
Yep a dollar should be 10**18, it doesn't work on my local env somehow, though the setup allocates 100M ether to test accounts, but somehow everyone of them only has 100.